internal/adapters/operation_flow/redis/redis_test.go

```diff
@@ -164,3 +167,77 @@ func TestNextOperationID(t *testing.T) {
 		}, 5*time.Second, 100*time.Millisecond)
 	})
 }
+
+func TestWatchOperationCancelationRequests(t *testing.T) {
+	schedulerName := uuid.New().String()
+	operationID := uuid.New().String()
+
+	t.Run("successfully receives the scheduler name and operation ID to cancel", func(t *testing.T) {
```

Comment:
@luizmiranda7 WDYT about adding a test case when it fails to `Unmarshal` the incoming message? it should not return any error nor send any request to the channel, right?

Reply:
Yeah, that's tricky to test... once there's no clear feedback to test